hyperspy/_components/lorentzian.py

```diff
@@ -16,76 +16,75 @@
 # You should have received a copy of the GNU General Public License
 # along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
 
-import numpy as np
+from hyperspy._components.expression import Expression
 
-from hyperspy.component import Component
+class Lorentzian(Expression):
 
-
-class Lorentzian(Component):
-
-    r"""Cauchy-Lorentz distribution (a.k.a. Lorentzian function) component
+    r"""Cauchy-Lorentz distribution (a.k.a. Lorentzian function) component.
 
     .. math::
 
-        f(x)=\frac{a}{\pi}\left[\frac{\gamma}{\left(x-x_{0}\right)^{2}+\gamma^{2}}\right]
-
-    +---------------------+-----------+
-    |     Parameter       | Attribute |
-    +---------------------+-----------+
-    +---------------------+-----------+
-    |      :math:`a`      |     A     |
-    +---------------------+-----------+
-    |    :math:`\gamma`   |   gamma   |
-    +---------------------+-----------+
-    |      :math:`x_0`    |  centre   |
-    +---------------------+-----------+
+        f(x)=\frac{A}{\pi}\left[\frac{\gamma}{\left(x-x_{0}\right)^{2}+\gamma^{2}}\right]
+
+    ================= ===========
```

Comment:
What is the reason to change the header of the table? I would leave this kind of changes to be discussed in another PR (#2148 should be a better place?) so it can be done consistently across all components.

Reply:
See discussion at #2148

Reply:
Similarly to other similar PRs, please use the convention already in place for the table and propose your suggestion in #2148.

Reply:
Done, also for  #2149 (logistic function).